map/lanelet2_extension/lib/utilities.cpp

```diff
@@ -61,12 +60,14 @@ void getContactingLanelets(
   const lanelet::BasicPoint2d search_point, std::vector<int> * contacting_lanelet_ids)
 {
   if (!lanelet_map) {
-    ROS_ERROR_STREAM("No lanelet map is set!");
+    std::cerr << "No lanelet map is set!" << std::endl;
+    ;
     return;
   }
 
   if (contacting_lanelet_ids == nullptr) {
-    ROS_ERROR_STREAM(__FUNCTION__ << " contacting_lanelet_ids is null pointer!");
+    std::cerr << __FUNCTION__ << " contacting_lanelet_ids is null pointer!" << std::endl;
+    ;
```

Comment:
What is this `;` for? There are the same lines in some files.

Reply:
Oh, I think I just messed up when I replaced ROS_ERROR with a replace command.